app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.kt

```diff
@@ -71,10 +71,10 @@ class FocusCrashException : Exception()
 // Therefore we ignore those violations for now.
 @Suppress("LargeClass", "TooManyFunctions")
 class UrlInputFragment :
-    LocaleAwareFragment(),
-    View.OnClickListener,
-    SharedPreferences.OnSharedPreferenceChangeListener,
-    CoroutineScope {
+        LocaleAwareFragment(),
+        View.OnClickListener,
+        SharedPreferences.OnSharedPreferenceChangeListener,
+        CoroutineScope {
```

Comment:
Should we change these spacings back to the way they were?

Reply:
Yes, removed